packages/gravity-ui-web/docs/coding-standards.md

```diff
@@ -0,0 +1,299 @@
+# Coding standards
+
+All code contributions are expected to adhere to these conventions. Where possible, we check and enforce these via linting tools. Code reviewers should also check that pull requests follow these standards.
+
+<!--
+  Tip: If using VSCode, get the "Auto Markdown TOC" extension by
+  Hunter Tran to automatically update this table of contents.
+  https://github.com/huntertran/markdown-toc
+-->
+<!-- TOC depthFrom:2 -->
+
+- [File formatting](#file-formatting)
+- [File names](#file-names)
+- [HTML](#html)
+- [SASS / CSS](#sass--css)
+    - [General](#general)
+    - [Declaration and property ordering](#declaration-and-property-ordering)
+    - [Selectors](#selectors)
+        - [Class names](#class-names)
+        - [Further reading](#further-reading)
+    - [SASS variables, functions and mixins](#sass-variables-functions-and-mixins)
+    - [Media queries](#media-queries)
+- [JavaScript](#javascript)
+    - [Browser JS](#browser-js)
+    - [Node.js build scripts](#nodejs-build-scripts)
+
+<!-- /TOC -->
+
+## File formatting
+All our source files must:
+
+* Use UNIX-style line endings (i.e. `\n`).
+* Indentation using 2 space characters.
+* Be UTF-8 encoded.
+* Have a blank line at the end.
+
+Note, we have an [`.editorconfig` file](https://editorconfig.org/) in the root of this repository. Most IDEs support this and will therefore apply the above rules automatically.
+
+## File names
+Source files should have lowercase, kebab-case filenames.
```

Comment:
Maybe we should use fs-lint to enforce this?

Reply:
I didn't even know such a thing existed. Sounds like a good idea. Mind raising an issue (or even PR) to do it?